docs/QSG/docker.md

```diff
@@ -1,172 +1,87 @@
 
 - The `daos-server` node running a DAOS server daemon managing data storage devices such as SCM or
   NVMe disks.
-- The `daos-admin` node allowing to manage the DAOS server thanks to `dmg`command.
-- The `daos-client` node using the the DAOS server to store data.
+- The `daos-admin` node is used to manage the DAOS server thanks to the `dmg`command.
+- The `daos-client` node uses the DAOS server to store data.
 
-At this time only emulated hardware storage are supported by this Docker platform:
+At this time only emulated hardware storage is supported by this Docker platform:
 
-- SCM (i.e. Storage Class Memory) are emulated with standard RAM memory.
+- SCM (i.e. Storage Class Memory) is emulated with standard RAM memory.
 - NVMe disks are emulated with a file device.
 
+!!!warning
+Virtual Docker networking is still in development. The following are known issues:
+- [bridge](https://docs.docker.com/network/bridge/) 
+- loopback
+	- Workaround - use one physical network interface for the [host](https://docs.docker.com/network/host/) used by the Docker containers network.
+
+## Hardware and Software Prerequisites
+### Hardware Prerequisites
+- 5GB of DDR and 
+- 16GB of storage.
+
+### Software Prerequisites
+- To build and deploy the Docker images, `docker` 
+	- optionally `docker-compose`
+- Hugepages must be enabled
+	- At least, 4096 pages of 2048kB should be available 
+- Internet access for the following repositories
+	- [Docker Hub](https://hub.docker.com/) 
+	- [Rocky Linux](https://rockylinux.org/) official repositories.  
+	- [hugepages](https://www.kernel.org/doc/Documentation/vm/hugetlbpage.txt) enabled
+		-Linux kernel feature shall be enabled on the docker host. At least, 4096 pages of 2048kB should be available. 
+
+### Hardware and software configuration used in this QSG
+- Server Board: Wolf Pass
+- CPU: Cascade Lake (2)
+- OS: Ubuntu 20.0.4LTE. freshly Installed
+- Memory: 192GB 
+- PMEM:(12)x16GB
+
+## Configuring Hugepages
+Additional information in Hugepages is located in the administration portion of this guide <need link> and at (kernel.org)[https://www.kernel.org/doc/Documentation/vm/hugetlbpage.txt]
+	
+### Determining Huge pages
+The number of huge pages allocated could be checked with the following command: 
+- Ubuntu- ```$ sysctl vm.nr_hugepages```
+
+The default size of a huge page, the number of available huge pages, etc. could be found with the following command: 
+- Ubuntu- ```$ cat /proc/meminfo | grep -e "^Huge" ```
+
+@@@ Not even sure what this means-The platform was tested and validated with the [rockylinux/rockylinux:8.4](https://hub.docker.com/r/rockylinux/rockylinux) and [centos:centos8](https://hub.docker.com/_/centos) official docker images.  However other RHEL-like distributions should be supported.
+
+@@@ Is this a warning about rocky and huge pages or daos with hugepages on rocky or??? 
```

Comment:
For an undefined reasons, it was not possible to use the rockylinux:8.5 as base image => error with huge page management when starting the daos-server. However, it was successfully tested with rockylinux:8.4 and centos:centos8 base docker images.

Reply:
commented